include/circt/Dialect/Calyx/CalyxLoweringUtils.h

```diff
@@ -33,6 +34,8 @@
 namespace circt {
 namespace calyx {
 
+using json = nlohmann::ordered_json;
```

Comment:
Does this dependency already exist in CIRCT?

Reply:
Oh, I see you've just created an additional header file. I don't think copying and pasting a third-party header file into CIRCT source code is a good idea. Is there a reason why this can't be done via some external API instead of introducing this dependency?

Reply:
Drive-by comment: I agree we should not copy and paste a third-party header to CIRCT. FYI ESI has an external dependency to nlohmann/json https://github.com/llvm/circt/blob/ffde8abb3a6c1848604ed7007be9d9943bb56228/lib/Dialect/ESI/runtime/CMakeLists.txt#L40 so you can do the similar thing. Also LLVM has a json library which is used extensively within CIRCT (e.g. https://github.com/llvm/circt/blob/ffde8abb3a6c1848604ed7007be9d9943bb56228/lib/Dialect/Arc/ModelInfo.cpp#L126-L129) so it would be great if we can use LLVM json library there is no particular reason.

Reply:
Thanks @uenoku !